morula.yml

```diff
@@ -1,5 +1,7 @@
 before-all: 'exosphere-shared'
 after-all: ""
 always: ""
-never: "documentation"
+never:
+  - documentation
+  - bin
```

Comment:
This will not work until I merge the PR that uses the latest version of Morula on Travis: #192